yarn.lock

```diff
@@ -447,11 +447,6 @@
   resolved "https://registry.yarnpkg.com/@xtuc/long/-/long-4.2.2.tgz#d291c6a4e97989b5c61d9acf396ae4fe133a718d"
```

Comment:
I think this file should not be updated

Reply:
Yes, it's a very strange. I've tried to drop node_modules folder in master and re-run yarn. I still have the same result, yarn.lock changed. Can you test on your side?

Reply:
What yarn version are you using?

Reply:
1.19.2

Reply:
The same behaviour with the latest 1.21.1

Reply:
The same for me with `rm -rf ./node_modules && yarn`